tarea 1/Luis Ernesto/App.java

```diff
@@ -0,0 +1,24 @@
+public class App {
+    public static void main(String[] args) throws Exception {
+       
+        String multiplos="";
+
+        for(int i=1;i<=200;i++){
+            if(i%5==0 && i%3==0){
+                multiplos=multiplos+"trescinco"+",\n";
```

Comment:
Todas estas líneas se pueden simplificar con el operador `+=`

```suggestion
                multiplos+="trescinco"+",\n";
```

Reply:
normalmente lo escribo asi para no confundirme cuando hay muchas variables 
por eso ya me acostumbre a hacerlo asi 
pero ya está corregido